### ISSUE TITLE
Stack policy handling for destroy when stack in ROLLBACK_COMPLETE state

### ISSUE THREAD

**Issue by jimcroft** [sparkleformation/sfn#120]
If a stack is destroyed by sfn it will attempt to deactivate the stack policy before tear down. Most of the time that's fine but modifying the stack policy while the stack is in ROLLBACK_COMPLETE state is a CloudFormation API error. You have to fall back to API / CLI / Console to delete the stack.

In addition stack policies aren't enforced for a full stack delete operation anyway. They only protect against resource modification during updates. So attempting to modify the policy prior to delete is unnecessary.

Ref: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/protect-stack-resources.html

```
Stack policies apply only during stack updates and should be used only as 
 fail-safe mechanism to prevent accidental updates to certain stack resources.
```

**Comment by chrisroberts** [comment]
:+1:

**Comment by chrisroberts** [comment]
Fix applied in 2.1.2. Thanks for the report!
